mitosheet/css/elements/MultiToggleBox.css

```diff
@@ -1,10 +1,10 @@
-.mutli-toggle-box-container {
+.multi-toggle-box-container {
```

Comment:
I prefer mutli. It sounds like a healthy brand of cereal.

Reply:
lol